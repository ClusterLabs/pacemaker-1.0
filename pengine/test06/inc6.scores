Allocation scores:
clone_color: rsc1 allocation score on node1: 0
clone_color: rsc1 allocation score on node2: 0
clone_color: child_rsc1:0 allocation score on node1: 1
clone_color: child_rsc1:0 allocation score on node2: 0
clone_color: child_rsc1:1 allocation score on node1: 0
clone_color: child_rsc1:1 allocation score on node2: 1
clone_color: child_rsc1:2 allocation score on node1: 0
clone_color: child_rsc1:2 allocation score on node2: 0
native_color: child_rsc1:0 allocation score on node1: 1
native_color: child_rsc1:0 allocation score on node2: 0
native_color: child_rsc1:1 allocation score on node1: -1000000
native_color: child_rsc1:1 allocation score on node2: 1
native_color: child_rsc1:2 allocation score on node1: -1000000
native_color: child_rsc1:2 allocation score on node2: -1000000
clone_color: rsc2 allocation score on node1: 0
clone_color: rsc2 allocation score on node2: 0
clone_color: child_rsc2:0 allocation score on node1: 1
clone_color: child_rsc2:0 allocation score on node2: 0
clone_color: child_rsc2:1 allocation score on node1: 1
clone_color: child_rsc2:1 allocation score on node2: 0
clone_color: child_rsc2:2 allocation score on node1: 0
clone_color: child_rsc2:2 allocation score on node2: 0
native_color: child_rsc2:0 allocation score on node1: 1
native_color: child_rsc2:0 allocation score on node2: 0
native_color: child_rsc2:1 allocation score on node1: -1000000
native_color: child_rsc2:1 allocation score on node2: 0
native_color: child_rsc2:2 allocation score on node1: -1000000
native_color: child_rsc2:2 allocation score on node2: -1000000
clone_color: rsc3 allocation score on node1: 0
clone_color: rsc3 allocation score on node2: 0
clone_color: child_rsc3:0 allocation score on node1: 1
clone_color: child_rsc3:0 allocation score on node2: 0
clone_color: child_rsc3:1 allocation score on node1: 0
clone_color: child_rsc3:1 allocation score on node2: 1
clone_color: child_rsc3:2 allocation score on node1: 0
clone_color: child_rsc3:2 allocation score on node2: 0
native_color: child_rsc3:0 allocation score on node1: 1
native_color: child_rsc3:0 allocation score on node2: 0
native_color: child_rsc3:1 allocation score on node1: -1000000
native_color: child_rsc3:1 allocation score on node2: 1
native_color: child_rsc3:2 allocation score on node1: -1000000
native_color: child_rsc3:2 allocation score on node2: -1000000
clone_color: rsc4 allocation score on node1: 0
clone_color: rsc4 allocation score on node2: 0
clone_color: child_rsc4:0 allocation score on node1: 1
clone_color: child_rsc4:0 allocation score on node2: 0
clone_color: child_rsc4:1 allocation score on node1: 1
clone_color: child_rsc4:1 allocation score on node2: 0
clone_color: child_rsc4:2 allocation score on node1: 0
clone_color: child_rsc4:2 allocation score on node2: 0
native_color: child_rsc4:0 allocation score on node1: 1
native_color: child_rsc4:0 allocation score on node2: 0
native_color: child_rsc4:1 allocation score on node1: -1000000
native_color: child_rsc4:1 allocation score on node2: 0
native_color: child_rsc4:2 allocation score on node1: -1000000
native_color: child_rsc4:2 allocation score on node2: -1000000
clone_color: rsc5 allocation score on node1: 0
clone_color: rsc5 allocation score on node2: 0
clone_color: child_rsc5:0 allocation score on node1: 0
clone_color: child_rsc5:0 allocation score on node2: 1
clone_color: child_rsc5:1 allocation score on node1: 0
clone_color: child_rsc5:1 allocation score on node2: 1
clone_color: child_rsc5:2 allocation score on node1: 0
clone_color: child_rsc5:2 allocation score on node2: 0
native_color: child_rsc5:0 allocation score on node1: 0
native_color: child_rsc5:0 allocation score on node2: 1
native_color: child_rsc5:1 allocation score on node1: 0
native_color: child_rsc5:1 allocation score on node2: -1000000
native_color: child_rsc5:2 allocation score on node1: -1000000
native_color: child_rsc5:2 allocation score on node2: -1000000
clone_color: rsc6 allocation score on node1: 0
clone_color: rsc6 allocation score on node2: 0
clone_color: child_rsc6:0 allocation score on node1: 1
clone_color: child_rsc6:0 allocation score on node2: 0
clone_color: child_rsc6:1 allocation score on node1: 0
clone_color: child_rsc6:1 allocation score on node2: 1
clone_color: child_rsc6:2 allocation score on node1: 0
clone_color: child_rsc6:2 allocation score on node2: 0
native_color: child_rsc6:0 allocation score on node1: 1
native_color: child_rsc6:0 allocation score on node2: 0
native_color: child_rsc6:1 allocation score on node1: -1000000
native_color: child_rsc6:1 allocation score on node2: 1
native_color: child_rsc6:2 allocation score on node1: -1000000
native_color: child_rsc6:2 allocation score on node2: -1000000
clone_color: rsc7 allocation score on node1: 0
clone_color: rsc7 allocation score on node2: 0
clone_color: child_rsc7:0 allocation score on node1: 0
clone_color: child_rsc7:0 allocation score on node2: 1
clone_color: child_rsc7:1 allocation score on node1: 0
clone_color: child_rsc7:1 allocation score on node2: 1
clone_color: child_rsc7:2 allocation score on node1: 0
clone_color: child_rsc7:2 allocation score on node2: 0
native_color: child_rsc7:0 allocation score on node1: 0
native_color: child_rsc7:0 allocation score on node2: 1
native_color: child_rsc7:1 allocation score on node1: 0
native_color: child_rsc7:1 allocation score on node2: -1000000
native_color: child_rsc7:2 allocation score on node1: -1000000
native_color: child_rsc7:2 allocation score on node2: -1000000
clone_color: rsc8 allocation score on node1: 0
clone_color: rsc8 allocation score on node2: 0
clone_color: child_rsc8:0 allocation score on node1: 1
clone_color: child_rsc8:0 allocation score on node2: 0
clone_color: child_rsc8:1 allocation score on node1: 0
clone_color: child_rsc8:1 allocation score on node2: 1
clone_color: child_rsc8:2 allocation score on node1: 0
clone_color: child_rsc8:2 allocation score on node2: 0
native_color: child_rsc8:0 allocation score on node1: 1
native_color: child_rsc8:0 allocation score on node2: 0
native_color: child_rsc8:1 allocation score on node1: -1000000
native_color: child_rsc8:1 allocation score on node2: 1
native_color: child_rsc8:2 allocation score on node1: -1000000
native_color: child_rsc8:2 allocation score on node2: -1000000
