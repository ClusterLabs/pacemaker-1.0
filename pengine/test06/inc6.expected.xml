<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="1" num_updates="1" dc-uuid="0" have-quorum="false" remote-tls-port="0" validate-with="pacemaker-1.0">
  <configuration>
    <crm_config><cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="no-quorum-policy" value="ignore"/></cluster_property_set></crm_config>
    <nodes>
      <node id="uuid1" uname="node1" type="member"/>
      <node id="uuid2" uname="node2" type="member"/>
    </nodes>
    <resources>
      <clone id="rsc1"><meta_attributes id="clone-rsc1.meta"><nvpair id="nvpair" name="resource-stickiness" value="1"/><nvpair id="nvpair" name="clone-max" value="3"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><primitive id="child_rsc1" class="heartbeat" type="apache"><meta_attributes id="primitive-child_rsc1.meta"/></primitive></clone>
      <clone id="rsc2"><meta_attributes id="clone-rsc2.meta"><nvpair id="restart_type.meta.auto-38" name="restart-type" value="restart"/><nvpair id="nvpair" name="resource-stickiness" value="1"/><nvpair id="nvpair" name="clone-max" value="3"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><primitive id="child_rsc2" class="heartbeat" type="apache"><meta_attributes id="primitive-child_rsc2.meta"/></primitive></clone>
      <clone id="rsc3"><meta_attributes id="clone-rsc3.meta"><nvpair id="restart_type.meta.auto-55" name="restart-type" value="restart"/><nvpair id="nvpair" name="resource-stickiness" value="1"/><nvpair id="nvpair" name="clone-max" value="3"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><primitive id="child_rsc3" class="heartbeat" type="apache"><meta_attributes id="primitive-child_rsc3.meta"/></primitive></clone>
      <clone id="rsc4"><meta_attributes id="clone-rsc4.meta"><nvpair id="nvpair" name="resource-stickiness" value="1"/><nvpair id="nvpair" name="clone-max" value="3"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><primitive id="child_rsc4" class="heartbeat" type="apache"><meta_attributes id="primitive-child_rsc4.meta"/></primitive></clone>
      <clone id="rsc5"><meta_attributes id="clone-rsc5.meta"><nvpair id="nvpair" name="resource-stickiness" value="1"/><nvpair id="nvpair" name="clone-max" value="3"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><primitive id="child_rsc5" class="heartbeat" type="apache"><meta_attributes id="primitive-child_rsc5.meta"/></primitive></clone>
      <clone id="rsc6"><meta_attributes id="clone-rsc6.meta"><nvpair id="restart_type.meta.auto-106" name="restart-type" value="restart"/><nvpair id="nvpair" name="resource-stickiness" value="1"/><nvpair id="nvpair" name="clone-max" value="3"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><primitive id="child_rsc6" class="heartbeat" type="apache"><meta_attributes id="primitive-child_rsc6.meta"/></primitive></clone>
      <clone id="rsc7"><meta_attributes id="clone-rsc7.meta"><nvpair id="restart_type.meta.auto-123" name="restart-type" value="restart"/><nvpair id="nvpair" name="resource-stickiness" value="1"/><nvpair id="nvpair" name="clone-max" value="3"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><primitive id="child_rsc7" class="heartbeat" type="apache"><meta_attributes id="primitive-child_rsc7.meta"/></primitive></clone>
      <clone id="rsc8"><meta_attributes id="clone-rsc8.meta"><nvpair id="nvpair" name="resource-stickiness" value="1"/><nvpair id="nvpair" name="clone-max" value="3"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><primitive id="child_rsc8" class="heartbeat" type="apache"><meta_attributes id="primitive-child_rsc8.meta"/></primitive></clone>
    </resources>
    <constraints>
      <rsc_order id="order1" first="rsc1" then="rsc2" score="0"/>
      <rsc_order id="order2" first="rsc3" then="rsc4" score="0"/>
      <rsc_order id="order3" first="rsc5" then="rsc6" score="0"/>
      <rsc_order id="order4" first="rsc7" then="rsc8" score="0"/>
    </constraints>
  </configuration>
  <status>
    <node_state id="uuid1" ha="active" uname="node1" crmd="online" join="member" expected="member" in_ccm="true">
      <lrm id="any">
        <lrm_resources id="lrm_resources">
          <lrm_resource id="rsc1:child_rsc1:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc1-child_rsc1-0_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc2:child_rsc2:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc2-child_rsc2-0_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc2:child_rsc2:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc2-child_rsc2-1_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc3:child_rsc3:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc3-child_rsc3-0_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc4:child_rsc4:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc4-child_rsc4-0_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc4:child_rsc4:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc4-child_rsc4-1_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc6:child_rsc6:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc6-child_rsc6-0_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc8:child_rsc8:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc8-child_rsc8-0_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>

          <lrm_resource id="rsc1:child_rsc1:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc1-child_rsc1-0_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc1:child_rsc1:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc1-child_rsc1-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc1:child_rsc1:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc1-child_rsc1-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc3:child_rsc3:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc3-child_rsc3-0_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc3:child_rsc3:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc3-child_rsc3-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc3:child_rsc3:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc3-child_rsc3-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc5:child_rsc5:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc5-child_rsc5-0_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc5:child_rsc5:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc5-child_rsc5-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc5:child_rsc5:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc5-child_rsc5-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc7:child_rsc7:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc7-child_rsc7-0_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc7:child_rsc7:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc7-child_rsc7-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc7:child_rsc7:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc7-child_rsc7-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc2:child_rsc2:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc2-child_rsc2-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc2:child_rsc2:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc2-child_rsc2-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc4:child_rsc4:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc4-child_rsc4-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc4:child_rsc4:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc4-child_rsc4-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc6:child_rsc6:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc6-child_rsc6-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc6:child_rsc6:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc6-child_rsc6-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc8:child_rsc8:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc8-child_rsc8-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc8:child_rsc8:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc8-child_rsc8-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>

        </lrm_resources>
      </lrm>
    </node_state>
    <node_state id="uuid2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true">
      <lrm id="any">
        <lrm_resources id="lrm_resources">
          <lrm_resource id="rsc1:child_rsc1:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc1-child_rsc1-1_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc3:child_rsc3:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc3-child_rsc3-1_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc5:child_rsc5:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc5-child_rsc5-0_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc5:child_rsc5:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc5-child_rsc5-1_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc6:child_rsc6:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc6-child_rsc6-1_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc7:child_rsc7:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc7-child_rsc7-0_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc7:child_rsc7:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc7-child_rsc7-1_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc8:child_rsc8:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc8-child_rsc8-1_start_0" operation="start" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="0" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>

          <lrm_resource id="rsc1:child_rsc1:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc1-child_rsc1-0_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc1:child_rsc1:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc1-child_rsc1-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc1:child_rsc1:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc1-child_rsc1-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc2:child_rsc2:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc2-child_rsc2-0_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc2:child_rsc2:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc2-child_rsc2-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc2:child_rsc2:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc2-child_rsc2-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc3:child_rsc3:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc3-child_rsc3-0_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc3:child_rsc3:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc3-child_rsc3-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc3:child_rsc3:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc3-child_rsc3-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc4:child_rsc4:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc4-child_rsc4-0_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc4:child_rsc4:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc4-child_rsc4-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc4:child_rsc4:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc4-child_rsc4-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc5:child_rsc5:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc5-child_rsc5-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc6:child_rsc6:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc6-child_rsc6-0_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc6:child_rsc6:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc6-child_rsc6-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc6:child_rsc6:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc6-child_rsc6-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc7:child_rsc7:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc7-child_rsc7-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc8:child_rsc8:0" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc8-child_rsc8-0_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc8:child_rsc8:1" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc8-child_rsc8-1_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>
          <lrm_resource id="rsc8:child_rsc8:2" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc8-child_rsc8-2_monitor_0" operation="monitor" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" interval="0" op-status="0" rc-code="7" call-id="1" crm_feature_set="1.0.6" transition-magic=""/>
	  </lrm_resource>

        </lrm_resources>
      </lrm>
    </node_state>
  </status>
</cib>
